ui/raidboss/raidboss_config.js

```diff
@@ -1219,7 +1219,7 @@ UserConfig.registerOptions('raidboss', {
         fr: 'Utiliser le navigateur pour la synthèse vocale', // Machine Translation
         ko: '웹브라우저를 이용해서 TTS 작동시키기',
         ja: 'ブラウザでTTS',
-        cn: '使用浏览器进行文字转语音', // Machine translation
+        cn: '忽略ACT的设置，使用Cactbot自带的Google TTS功能（需联网）',
```

Comment:
So it may not be usable in Mainland China

Reply:
Well Google is banned in China ><